server/test/estree-utils-test.ts

```diff
@@ -1,4 +1,4 @@
-import { expect } from 'chai';
+const { expect } = require('chai');
```

Comment:
because using `noImplicitAny` means that imports need to be typed and we don't have types for `chai`